frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -761,15 +784,73 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
   );
 });
 
+const InitializationResourceAlert: React.SFC<InitializationResourceAlertProps> = (props) => {
+  const { initializationResource, csv } = props;
+
+  const initializationResourceKind = initializationResource?.kind;
+  const initializationResourceDefinition = props.csv.spec?.customresourcedefinitions?.owned?.find(
+    (api) => api.kind === initializationResourceKind,
+  );
+  const initializationResourceModelName = initializationResourceDefinition?.name;
+  const { group: initializationResourceGroup } = groupVersionFor(
+    initializationResource?.apiVersion,
+  );
+  const { findModel } = useModelFinder();
+  const model = findModel(
+    initializationResourceGroup,
+    initializationResourceModelName.split('.').shift(),
+  );
```

Comment:
You have the sample resource, right? Can you just call `referenceFor(initializationResource)` followed by `modelFor`? Then you don't need `useModelFinder`.